.github/workflows/rw-publish-version.yml

```diff
@@ -1,36 +1,41 @@
+# (C) 2024 GoodData Corporation
+
 name: rw ~ Rush ~ Publish version
-description: 'TODO'
 on:
-  workflow_call:
-    inputs:
-      git-hub-token:
-        description: 'GitHub token with write access to the source branch'
-        required: true
-      source-branch:
-        required: true
-        description: 'The name of the source branch'
-        type: string
+    workflow_call:
+        inputs:
+            source-branch:
+                required: true
+                description: "The name of the source branch"
+                type: string
+            release-tag:
```

Comment:
I don't see this used below, is it forgotten or maybe I'm missing something

Reply:
Its prepared for release for dry run is not necessary